modules/bs.environment/data/bs.environment/function/celestial/look_at_moon.mcfunction

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------------------------------------------------
+# Copyright (c) 2025 Gunivers
+#
+# This file is part of the Bookshelf project (https://github.com/mcbookshelf/bookshelf).
+#
+# This source code is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Conditions:
+# - You may use this file in compliance with the MPL v2.0
+# - Any modifications must be documented and disclosed under the same license
+#
+# For more details, refer to the MPL v2.0.
+# ------------------------------------------------------------------------------------------------------------
+
+execute store result storage bs:ctx y float 0.001 run function bs.environment:celestial/get_moon_angle
+execute store result score #t bs.ctx run time query daytime
```

Comment:
Maybe this command can be avoided by reusing the one in get angle. But that's not super clean so unsure

Reply:
I propose to split "get_X_angle" into two different features:

- get_current_X_angle, which will have the same behavior
- get_X_angle, which proposes to specify the daytime.

loot_at_X will basically call the second one to avoid this getting daytime duplication.
What do you think?